### PR TITLE
translatesfx: put process list receiver in logs pipeline

### PR DESCRIPTION
**Pull request by pmcollins**

### CODE REVIEW

**Comment by dmitryax** [comment]
Please rebase

**Comment by pmcollins** [comment]
> Please rebase

Done

File: cmd/translatesfx/translatesfx/otel.go [hunk 1]
```diff
@@ -25,6 +25,8 @@ type otelCfg struct {
 	Service       map[string]interface{}
 }
 
+const processlist = "smartagent/processlist"
```
Comment (rmfitzpatrick) [inline cmd/translatesfx/translatesfx/otel.go:28]: This should also include `kubernetes-events` but that can be another PR.  Some others should be in both metrics and logs (https://github.com/signalfx/splunk-otel-collector/blob/main/internal/receiver/smartagentreceiver/README.md#configuration  item 6)

**Review by rmfitzpatrick — APPROVED** [review]
Thanks!
